sources/sections-en/05-unit-symbols.adoc

```diff
@@ -183,7 +188,7 @@ Quantities that are ratios of quantities of the same kind (for example length ra
 
 The internationally recognized symbol % (percent) may be used with the SI. When it is used, a space separates the number and the symbol %. The symbol % should be used rather than the name "percent". In written text, however, the symbol % generally takes the meaning of "parts per hundred". Phrases such as "percentage by mass", "percentage by volume", or "percentage by ((amount of substance))" shall not be used; the extra information on the quantity should instead be conveyed in the description and symbol for the quantity.
 
-The term "((ppm))", meaning stem:[10^(−6)] relative value, or 1 part in stem:[10^6], or parts per million, is also used. This is analogous to the meaning of percent as parts per hundred. The terms "parts per billion" and "parts per trillion" and their respective abbreviations "((ppb))" and "((ppt))", are also used, but their meanings are language dependent. For this reason the abbreviations ppb and ppt should be avoided.
+The term "((ppm))", meaning stem:[10^(−6)] relative value, or 1 part in stem:[10^6], or parts per million, is also used. This is analogous to the meaning of percent as parts per hundred. The terms "parts per billion" and "parts per trillion" and their respective abbreviations "((ppb))" and "((ppt))"&#x200c;, are also used, but their meanings are language dependent. For this reason the abbreviations ppb and ppt should be avoided.
```

Comment:
I think we don't need to put something as workaround solution for obvious bug.

Reply:
Removed. Thanks!